### DISCUSSION TITLE
Is the source code of mermaid diagrams recoverable after importing?

### DISCUSSION THREAD

**Discussion by jorgegomezcq** [jgraph/drawio#3942] [Q&A]
Hi,

Is the source code of mermaid diagrams recoverable after importing one?

It looks like draw.io converted it to svg but now I can't go back and edit the mermaid source code for the diagram.

I don't have access to the original source code after importing the mermaid diagram into drawio.

**Comment by davidjgraph** [comment]
When inserting mermaid you select whether or not insert as an image (and can then further edit the mermaid input), or convert to draw.io natively. If you create as draw.io you cannot edit the mermaid input.
